src/endpoints/Endpoint.js

```diff
@@ -53,22 +49,23 @@ class Endpoint {
   /**
    * @param {string} name The name of this endpoint. This is used as key in the state and as Payload name. The Payload name is name + 'Paylaod'
    * @param {string} url The url with params (params are used like this: https://cms.integreat-app.de/{location}/{language})
-   * @param {function} jsonToAny Transforms the json input to a result
-   * @param {mapStateToOptionsCallback} mapStateToUrlParams Maps the state to the url params which are needed in the Fetcher component
+   * @param {function} mapData Transforms the json input to a result
```

Comment:
mapData is too generic imo. Why not mapJsonResponse since this would also fit the javaDoc comment?

Reply:
done